.github/workflows/publish_container.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+      - health-check
```

Comment:
Oops. This PR branch was originally called health-check, because I was adding a health-check, and so I wanted to trigger the build in this branch, and I forgot to remove the branch from the GH actions workflow. But also, the DO deploy builds its own container now anyway.